tests/robot-tests/tests/libs/public-common.robot

```diff
@@ -43,3 +43,12 @@ user checks methodology note
     [Arguments]    ${number}    ${displayDate}    ${content}
     user waits until element contains    css:#methodologyNotes li:nth-of-type(${number}) time    ${displayDate}
     user waits until element contains    css:#methodologyNotes li:nth-of-type(${number}) p    ${content}
+
+user verifies methodology is visible on public methodology page
+    [Arguments]    ${THEME}    ${TOPIC}    ${METHODOLOGY_NAME}    ${METHODOLOGY_SLUG}
+    user navigates to public methodologies page
+    user waits until page contains accordion section    ${THEME}
+    ${accordion}=    user opens accordion section    ${THEME}
+    ${details}=    user opens details dropdown    ${TOPIC}    ${accordion}
+    user checks page contains element    xpath://*[a/text()="${METHODOLOGY_NAME}"]
```

Comment:
I think they are listed within their publication and this doesn't seem to check the publication name so they could be appearing in any publication. I think where we do this kind of check already in `bau\publish_methodology.robot` that we check the publication name as well as the methodology name.